plugins/out_azure_kusto/azure_kusto_conf.c

```diff
@@ -205,136 +194,139 @@ static int parse_storage_resources(struct flb_azure_kusto *ctx, struct flb_confi
 
     jsmn_init(&parser);
     tokens = flb_calloc(1, sizeof(jsmntok_t) * tok_size);
```

Comment:
Is there a specific reason to manually multiply the element size by the element count here instead of passing `tok_size` as first argument to `flb_calloc`?

Reply:
TBH, I just saw that in other plugins that parsed JSON, thought it was weird too, but assumed it was for a good reason, and decided to do the same.